### DISCUSSION TITLE
osu!taikomania Combo

### DISCUSSION THREAD

**Discussion by RichardKanshen** [ppy/osu#14156] [Ideas] [answered]
osu!taiko and osu!mania are pretty much opposites, aren't they? What about adding a new game mode that can be triggered by clicking on taiko and then mania (or reverse)? Single game mode could be triggered by clicking osu!/osu!catch or other game mode and then the other one.

Here is the idea: osu! mania part would stick left wall of window, with shortened upper part for osu!taiko. This game mode would **COMBINE** osu!mania and osu!taiko.

**Comment by bdach** [comment, marked as answer]
I have zero clue what this is proposing or what this "taikomania" would look like, sorry. Please explain this "proposal" better.

And *if* it happens (which it won't, most likely, at least not as an official ruleset) it should sure as hell not work like described above.

**Reply by Rekunan** [reply]
To make an attempt, he's proposing to have a separate taikomania ruleset. In his words it's meant to combine the two rulesets, but taikomania is basically changing the taiko field to be like 2k mania where one column is red, and the other is blue, which immensely helps with reading.

In the past, there was a skin that did this, and you'd rotate your screen/window to achieve the desired effect. Here's the [forum post](https://osu.ppy.sh/community/forums/topics/404353) for it and the [example video](https://www.youtube.com/watch?v=K_h4Tf32tII).

**Comment by TheFearlessDeath500** [comment]
taikomania is considered cheating so no

**Reply by Rekunan** [reply]
Well it's only considered cheating in the sense of taiko itself, but since it's clearly not base taiko, it wouldn't be cheating.
